jenkins/agent-dind.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
     labels:
         containers: dind
 spec:
+    nodeSelector:
+      eks.amazonaws.com/nodegroup: eks-oss-presto-dynamic-managed-ng
```

Comment:
What does this do?

Reply:
Linsong reconfigured the eks, created a dedicated node group for Jenkins to scale agents.